acados/ocp_nlp/ocp_nlp_sqp_rti.c

```diff
@@ -948,6 +948,8 @@ int ocp_nlp_sqp_rti(void *config_, void *dims_, void *nlp_in_, void *nlp_out_,
         config->dynamics[ii]->memory_set_ux1_ptr(nlp_out->ux + ii + 1, mem->dynamics[ii]);
         config->dynamics[ii]->memory_set_pi_ptr(nlp_out->pi + ii, mem->dynamics[ii]);
         config->dynamics[ii]->memory_set_BAbt_ptr(work->qp_in->BAbt + ii, mem->dynamics[ii]);
+        config->dynamics[ii]->memory_set_RSQrq_ptr(work->qp_in->RSQrq + ii, mem->dynamics[ii]);
```

Comment:
Your regularization has been added only to the SQP solver, but not to the SQP_RTI solver. It may be boring to do it twice, but is is only a matter of copy-and-paste. A better sharing of code between SQP and SQP_RTI will be implemented in the future.

Reply:
This kind of copy-pasting is dangerous. I added it, but use at your own risk.